.github/dependabot.yml

```diff
@@ -6,3 +6,9 @@ updates:
     schedule:
       # Check for updates to GitHub Actions every week
       interval: "weekly"
+  # Maintain dependencies for Cargo
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
```

Comment:
nit: this comment is copy-pasted from the previous block and doesn't apply here, IMO it can just be dropped